### DISCUSSION TITLE
Conditional RAVE?

### DISCUSSION THREAD

**Discussion by yeswecan** [acids-ircam/RAVE#313] [General]
Hello everyone!

I've stumbled upon a fairly old paper called ["Conditional variational autoencoder to improve neural audio synthesis for
polyphonic music sound"](https://www.semanticscholar.org/reader/4a91af2a5a4759594a92f2ed82763ba31bc945ea). The authors modify RAVE to condition it and improve pitch performance. There's no code though, but I wonder if anyone has tried to implement this or something like this, or if anyone interested in trying to replicate that?

Maybe there are concerns about the paper or w.e.? I'll maybe try to do it myself, if anyone wants to collaborate, hmu too!

**Comment by mukut03** [comment]
Hi - came across the same paper too. Would like to collaborate. Shoot me an email at mukutm2@Illinois.edu.
